packages/web-components/src/components/button/__stories__/README.stories.mdx

```diff
@@ -24,6 +24,14 @@ Here's a quick example to get you started.
 import '@carbon/ibmdotcom-web-components/es/components/button/button';
 ```
 
+or with [JSPM](https://jspm.org)
+
+```html
+<script type="module">
+  import 'https://jspm.dev/@carbon/ibmdotcom-web-components@latest/es/components/button/button.js';
```

Comment:
```suggestion
  import 'https://jspm.dev/@carbon/ibmdotcom-web-components@canary/es/components/button/button.js';
```

`@latest` tag doesn't work as of now, given `0.2.0` lacks `exports` field in `package.json`.